src/libcollections/str.rs

```diff
@@ -10,12 +10,15 @@
 //
 // ignore-lexer-test FIXME #15679
 
-//! Unicode string manipulation (the `str` type).
+//! Unicode string manipulation (the [`str`](../primitive.str.html) type).
 //!
-//! Rust's `str` type is one of the core primitive types of the language. `&str` is the borrowed
-//! string type. This type of string can only be created from other strings, unless it is a static
-//! string (see below). As the word "borrowed" implies, this type of string is owned elsewhere, and
-//! this string cannot be moved out of.
+//! Rust's [`str`](../primitive.str.html) type is one of the core primitive types of the language.
+//! `&str` is the borrowed string type. This type of string can only be created from other strings,
+//! unless it is a static string (see below). As the word "borrowed" implies, this type of string
+//! is owned elsewhere, and this string cannot be moved out of.
```

Comment:
Yes, it's totally fine to toss in small edits if you'd like.

I would actually replace 'static string' with `&'static str` here, to be more specific. 

And I like the suggested re-wording of that other sentence :+1: